### ISSUE TITLE
[bug] apps - aliyun app crash causes last timestamp to never get updated

### ISSUE THREAD

**Issue by ryandeivert** [airbnb/streamalert#926]
## Background

This error occurs:
```
Ending last timestamp is the same as the beginning last timestamp. This could occur if there were no logs collected for this execution.
```

After the following traceback:
```Traceback (most recent call last):
File "/var/task/stream_alert/apps/_apps/aliyun.py", line 81, in _gather_logs
response = self.client.do_action_with_exception(self.request)
File "/var/task/aliyunsdkcore/client.py", line 258, in do_action_with_exception
status, headers, body = self.implementation_of_do_action(acs_request)
File "/var/task/aliyunsdkcore/client.py", line 235, in implementation_of_do_action
error_msg.get_msg('SDK_SERVER_UNREACHABLE') + ': ' + str(e))
ClientException: SDK.ServerUnreachable Unable to connect server: timed out```

**Comment by chunyong-lin** [comment]
There was [a PR merged early this year to implement retry and timeout to aliyuncore code](https://github.com/aliyun/aliyun-openapi-python-sdk/pull/163/files). Most likely it will fix this issue.

I am going to upgrade `aliyun-python-sdk-core` sdk to latest version `2.13.5`.

**Comment by chunyong-lin** [comment]
This bug has been fixed in PR #970. 💯
